src/_tutorials/language/streams.md

```diff
@@ -1,22 +1,19 @@
 ---
 title: "Asynchronous Programming: Streams"
-description: "Learn how to consume single-subscriber and broadcast streams."
-
+description: Learn how to consume single-subscriber and broadcast streams.
 prevpage:
   url: /tutorials/language/futures
   title: "Asynchronous Programming: Futures"
 ---
 
 <div class="panel" markdown="1">
+  <h4>What's the point?</h4>
```

Comment:
Why `<h4>` instead of `####`?

Reply:
So that we can have it all nicely indented under the `<div>`. I find that it makes the entire file easier to skim and read.